packages/core/src/components/Box/Box.stories.mdx

```diff
@@ -0,0 +1,141 @@
+import { Box } from './Box.tsx';
+import { fontVariants, shadows } from './Box.stories';
+import { withKnobs, color, select, boolean, number } from '@storybook/addon-knobs';
+import { Preview, Meta, Props, Story } from '@storybook/addon-docs/blocks';
+import { defaultTheme } from '@medly-components/theme';
+
+<Meta title="Core" component={Box} />
+
+# Box
+
+A simple box component with customizable properties.
+
+You can play with the props in the **Canvas** tab.
```

Comment:
This line can be removed since user will already know their way around the storybook
```suggestion
```

Reply:
That's assuming the user already know how to use Canvas tab but incase someone is new then it will be helpful.
Also same is there for every core component in such as [Button Component](https://www.medlycomponents.com/?path=/docs/core--button)